code/modules/projectiles/guns/magic.dm

```diff
@@ -6,25 +6,27 @@
 	item_state = "staff"
 	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
+
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi' //not really a gun and some toys use these inhands
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```

Comment:
I got very confused by this and thought I should move this up to make it obvious to people that this is FUCKED and needs resolve. I don't know what the actual icon you guys want to use should be, I'm leaving it here as a more obvious actual problem so someone who cares can fix it.